compiler/executable/pipeline.rs

```diff
@@ -59,8 +59,8 @@ impl ExecutableStage {
             ExecutableStage::Insert(executable) => executable
                 .output_row_schema
                 .iter()
-                .filter_map(|opt| opt.map(|(v, _)| v))
                 .enumerate()
+                .filter_map(|(i, opt)| opt.map(|(v, _)| (i,v)))
```

Comment:
`(i,v)` -> `(i, v)`, let's show good formatting examples!

Btw, it's a good practice to run (I assume that it should fix this) 
`bazel run @rules_rust//:rustfmt --@rules_rust//:rustfmt.toml=//:rustfmt.toml` before you commit, you can also set it as a pre-commit hook. We'll add the CI/CD formatting check which will bonk your head if you don't do it.

Reply:
thanks for this!